include/tss2/xaptum_nvram.h

```diff
@@ -0,0 +1,56 @@
+/******************************************************************************
```

Comment:
Yea, I agree. I started doing that, and stopped because I didn't want to wade into creating two libs. But just separating the headers for now works for me.